third_party/libosmium/include/osmium/io/detail/pbf_decoder.hpp

```diff
@@ -105,6 +105,8 @@ namespace osmium {
                     while (pbf_string_table.next(OSMFormat::StringTable::repeated_bytes_s, protozero::pbf_wire_type::length_delimited)) {
                         const auto str_view = pbf_string_table.get_view();
                         if (str_view.size() > osmium::max_osm_string_length) {
+                            std::cout<< "### Overlong string error(max-size, overlong-string-size, overlong-string-content)" << str_view.size() << " "
```

Comment:
Does `libosmium` have better way to output error logs instead of `std::cout?

Reply:
#1. Error message should follow style of certain project.
#2. OSMIUM is a C++ project with very few dependencies, which is using exceptions for unexpected situation.
    Will put the information needed in the exception context.